js/ui/templates/site.es6.js

```diff
@@ -1,5 +1,6 @@
 const bel = require('bel');
 const toggleButton = require('./shared/toggle-button');
+const Button = require('./shared/button');
```

Comment:
`shared/button` is a function not a constructor, so it should be `const button` not `const Button`.